.github/workflows/pipeline.yml

```diff
@@ -130,6 +130,13 @@ jobs:
                   node-version: 14.x
             - name: Install pnpm modules
               run: pnpm install
+            - name: Replace placeholder with instrumentation key
+              uses: jacobtomlinson/gha-find-replace@v2
+              with:
+                  include: packages/ide-extension/src/telemetry/telemetry.ts
+                  find: 'ApplicationInsightsInstrumentationKeyPLACEH0LDER'
```

Comment:
Is it on purpose zero instead of o in ```PLACEH0LDER``` ?

Reply:
Yes, to minimize the risk of any conflicts ;).